sdk/include/assembly-helpers.h

```diff
@@ -21,6 +21,25 @@ struct CheckSize
 	static constexpr bool value = Real == Expected;
 };
 
+/**
+ * Export a macro into assembly named `name` with value `value`.  In C++, this
+ * macro will report an error if the provided value does not equal the constexpr
+ * evaluation of `expression`.
+ */
+#	define EXPORT_ASSEMBLY_NAME(name, val)                                    \
```

Comment:
It took me a minute to work out why we need both of these macros. I guess this one is just the same as `EXPORT_ASSEMBLY_EXPRESSION(name, name, val)` but without a name collision? That makes me think maybe we should use `{}` around the definition below?

Reply:
We might want the names available in C++, maybe, and the invoking source could, in principle, choose to put things in scope to hide them if desired, but because these are otherwise at top-level, just using `{}` doesn't do the trick, eliciting "expected unqualified-id" errors from the compiler.